foundations/foundation-config/src/main/java/org/apache/servicecomb/config/priority/ConfigObjectFactory.java

```diff
@@ -95,7 +100,8 @@ public List<ConfigObjectProperty> createProperties(Object instance, String prefi
         continue;
       }
 
-      Setter<Object, Object> setter = createObjectSetter(propertyDefinition);
+      Setter<Object, Object> setter = beanDescriptionCache.computeIfAbsent(propertyDefinition,
+              LambdaMetafactoryUtils::createObjectSetter);
```

Comment:
import static org.apache.servicecomb.foundation.common.utils.LambdaMetafactoryUtils.createObjectSetter  can be removed